schedule/jeos/sle/hyperv/jeos-main.yaml

```diff
@@ -49,4 +49,4 @@ schedule:
   - console/zypper_lifecycle
   - console/repo_orphaned_packages_check
   - console/orphaned_packages_check
-  - console/consoletest_finish
\ No newline at end of file
+  - console/consoletest_finish
```

Comment:
What is the change here? Both old and new versions of the file don't seem to have a new line at the end. Maybe you can checkout the file according to an old commit and commit again, if you want your PR to look clean.

Reply:
I like very much the idea to have a clean view. This is due to have or not have line break at the end of a yaml file, which is something that some editors do automatically but not others. If you run some kind of tool that adjust this for all files I'd suggest to do it in a different commit, otherwise is more clear only run it for your changes.

Reply:
those are found and i though to fix them as they are simple changes. i could have another commit but i think it s fine.

Reply:
it is fine, but it doesn't make easy to review your code and at least two people agree on that :) Just consider it, not that you have to change.

Reply:
i will try to put them in another commit but feel free to merge it in the meantime if this take time and ofc i will consider to have separate commit for these kind of things. it is better practice anyway

Reply:
second commit added